README.md

```
@@ -245,6 +261,12 @@ Collection returned on endpoint: /api/logs/withCollection, size: 2, status: 200
 
 ### Changelog:
 
+##### 20.07.2026
```

markdownlint-cli2 (0.23.0)

[warning] Heading levels should only increment by one level at a time
Expected: h4; Actual: h5

(MD001, heading-increment)